assets/js/base/components/price-slider/index.js

```diff
@@ -187,15 +189,24 @@ const PriceSlider = ( {
 	 */
 	const priceInputOnBlur = useCallback(
 		( event ) => {
+			// Only refresh when finished editing the min and max fields.
```

Comment:
I added this block to prevent queries running when tabbing between the two pricing fields which I noticed when testing!

Reply:
good catch.